lib/bitcoin/fee/SlidingWindowQuantileCalculator.ts

```diff
@@ -30,12 +30,14 @@ export default class SlidingWindowQuantileCalculator {
   /**
    * The value to use as an input to the ValueApproximator.
    */
-  private readonly feeApproximate: number = 1.1;
+  private readonly feeApproximate: number = 1.414;
   /**
    * Size of a frequency vector; 1 + max normalized value
    */
   private frequencyVectorSize: number;
 
+  private maxQuantileDeviationPercentageValue: number;
```

Comment:
I guess you took it out as a member variable for testing purposes.

Reply:
That and I wanted to divide by 100 and save the value as member.